editor/filesystem_dock.cpp

```diff
@@ -2512,6 +2520,7 @@ void FileSystemDock::_file_and_folders_fill_popup(PopupMenu *p_popup, Vector<Str
 			filenames.push_back(fpath);
 			all_folders = false;
 			all_files_scenes &= (EditorFileSystem::get_singleton()->get_file_type(fpath) == "PackedScene");
```

Comment:
Maybe store the file type in a local const (like here https://github.com/godotengine/godot/pull/70805/files#diff-5cfc95a5fb43c0bf71052045ddce94275819d0173bc0c2c7fcd667ffe2568ce2R2535)

Reply:
It can't be a const because it is being iteratively modified in the for loop.

Reply:
Thanks, that is a good suggestion. I added it 👍